### ISSUE TITLE
Loss funtion of nn4.small2.lrn.h5 ?

### ISSUE THREAD

**Issue by NguyenThaiHoc1** [iwantooxxoox/keras-facenet#13]
How to know loss function of model above ???

I don't know it. I want to know it 

Does anybody know ?
Please help me
